charger/eebus.go

```diff
@@ -293,17 +336,37 @@ func (c *EEBus) Enabled() (bool, error) {
 		return true, nil
 	}
 
-	limits, err := c.emobility.EVLoadControlObligationLimits()
+	// if the VW VAS PV mode is active, use PV limits
+	if c.hasActiveVASVW() {
+		limits, err := c.uc.OscEV.LoadControlLimits(evEntity)
+		if err != nil {
+			// there are no limits available, e.g. because the data was not received yet
+			return true, nil
+		}
+
+		for _, limit := range limits {
+			// check if there is an active limit set
+			if limit.IsActive && limit.Value >= 1 {
+				return true, nil
+			}
+		}
+
+		return false, nil
+	}
+
+	limits, err := c.uc.OpEV.LoadControlLimits(evEntity)
 	if err != nil {
-		// there are no overload protection limits available, e.g. because the data was not received yet
+		// there are limits available, e.g. because the data was not received yet
```

Comment:
@DerAndereAndi der Kommentar sollte "no limits" heißen? Oder ist die Bedingung falsch?

Reply:
Den Kommentar gibt es im aktuellen Code nicht und die Stelle greift nur bei VAS => Taycan. Das läuft sauber.

Reply:
Arg, falsche Stelle. Den gibts immer noch:

```go
	limits, err := c.uc.OpEV.LoadControlLimits(evEntity)
	if err != nil {
		// there are limits available, e.g. because the data was not received yet
		c.log.DEBUG.Println("!! OpEV.LoadControlLimits error", err)
		return true, nil
	}
```

Reply:
An der Stelle hier falsch?

Reply:
Glaube ich kaum, denn Limits sind im Ablauf ja da und gesetzt.

Reply:
Mir gehts nur um den Kommentar- der müsste heissen "no limits"? Dann würde ich das lokal korrigieren.